physicsnemo/models/gnn_layers/utils.py

```diff
@@ -450,6 +520,57 @@ def agg_concat_dgl(
         return cat_feat
 
 
+@torch.jit.ignore()
+def agg_concat_hetero_dgl(
```

Comment:
This and `concat_efeat_hetero_dgl` seem like possible to create PyG versions. I can take care of that once PR is merged.